src/pymovements/datasets/gazebasevr.py

```diff
@@ -196,29 +196,30 @@ class GazeBaseVR(DatasetDefinition):
         default_factory=lambda: {
             'xT': 'x_target_pos',
             'yT': 'y_target_pos',
+            'zT': 'z_target_pos',
         },
     )
 
     custom_read_kwargs: dict[str, dict[str, Any]] = field(
         default_factory=lambda: {
             'gaze': {
                 'schema_overrides': {
-                    'n': pl.Float32,
-                    'x': pl.Float32,
-                    'y': pl.Float32,
-                    'lx': pl.Float32,
-                    'ly': pl.Float32,
-                    'rx': pl.Float32,
-                    'ry': pl.Float32,
-                    'xT': pl.Float32,
-                    'yT': pl.Float32,
-                    'zT': pl.Float32,
-                    'clx': pl.Float32,
-                    'cly': pl.Float32,
-                    'clz': pl.Float32,
-                    'crx': pl.Float32,
-                    'cry': pl.Float32,
-                    'crz': pl.Float32,
+                    'n': pl.Float64,
```

Comment:
why Float64?

Reply:
good that you ask. somehow I thought that every other definition has float64 specified, but that's obviously wrong. probably it won't make any difference on cpu, only on gpus.

I reverted the type changes.

I'll defer merging until after the next release.

Reply:
I was probably mislead when writing tests for from_csv in #1004 as polars seems to read Float64 by default